packages/plugin-truffle/src/deploy-beacon.ts

```diff
@@ -15,8 +15,8 @@ export async function deployBeacon(Contract: ContractClass, opts: Options = {}):
   const UpgradeableBeaconFactory = getUpgradeableBeaconFactory(Contract);
   const beaconDeployment = await deploy(deployer, UpgradeableBeaconFactory, impl);
 
-  Contract.address = beaconDeployment.address;
-  const beaconContract = new Contract(beaconDeployment.address);
+  UpgradeableBeaconFactory.address = beaconDeployment.address;
```

Comment:
As far as I know this line won't have any effect. `deployed()` is only tracked for the user's contracts but the artifact for this is deep inside our code, not in the user's artifacts directory.